java/core/src/main/java/org/bondlib/protocol/UntaggedProtocolReader.java

```diff
@@ -193,6 +196,23 @@
     void skipWString() throws IOException;
 
     /**
+     * Skip based on supplied runtime schema, starting at the root of the SchemaDef
+     * 
+     * @param schema the schema def
+     * @throws IOException
+     */
+    void skip(SchemaDef schema) throws IOException;
```

Comment:
I prefer we don't have this method as it can trivially be implemented in terms of the more general method. Else, every implementation needs to implement an identical method (unless we use Java 8 with default implementation support).

Reply:
Done.